cli/tsc/00_typescript.js

```diff
@@ -51163,7 +51163,7 @@ var ts;
             var usageMode = file && getUsageModeForExpression(usage);
             if (file && usageMode !== undefined) {
                 var result = isESMFormatImportImportingCommonjsFormatFile(usageMode, file.impliedNodeFormat);
-                if (usageMode === ts.ModuleKind.ESNext || result) {
+                if (result) {
```

Comment:
This is unfortunate. Essentially typescript doesn't allow synthetic default imports when using `ModuleKind.ESNext`, which we need to do because deno now supports importing cjs. I'll open an issue in TypeScript tomorrow.